data_generator.py

```diff
@@ -0,0 +1,103 @@
+# Plantalytics
+#     Copyright (c) 2016 Sapphire Becker, Katy Brimm, Scott Ewing, 
+#       Matt Fraser, Kelly Ledford, Michael Limb, Steven Ngo, Eric Turley.
+#     This project is licensed under the MIT License.
+#     Please see the file LICENSE in this distribution for license terms.
+# Contact: plantalytics.capstone@gmail.com
+
+import ConfigParser
+import json
+import random
+import requests
+import sys
+import time
+
+# Read values from 
+def load_config():
+    config = ConfigParser.RawConfigParser()
+    config.read('hub.config')
+    # Load hub data
+    global HUB_ID 
+    HUB_ID = config.get('Hub Data', 'hub_id')
+    global VINE_ID
+    VINE_ID = config.get('Hub Data', 'vine_id')
+    global PUBLIC_KEY
+    PUBLIC_KEY = config.get('Hub Data', 'public_key')
```

Comment:
Heads up for the key: I've modified the endpoint to actually check this key for "auth". The key it's currently expecting is in our `env_var.txt file` in the doc repo. You'll want to make sure to populate `hub.config` with that value.